### PR TITLE
Update to OpenShift Pipelines 1.15

### PR DESCRIPTION
**Pull request by rnc**
Follows on from the 1.14 upgrade in #1915

### CODE REVIEW

**Comment by rnc** [comment]
I think the changes in https://github.com/redhat-appstudio/jvm-build-service/pull/1921 to update the controller-gen are needed (See also https://www.github.com/kubernetes-sigs/controller-tools/issues/880)

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/redhat-appstudio/jvm-build-service
 
-go 1.21
+go 1.22.0
```
Comment (stuartwdouglas) [inline go.mod:3]: Won't this also require an updated Dockerfile with the new go version?

Comment (rnc) [reply go.mod:3]: Ah, yes, I'd missed there was a go reference in the top level Dockerfile.

Comment (rnc) [reply go.mod:3]: Although, whether go1.22 is available is another question...

**Pull request merged**